pkg/shared/types.go

```diff
@@ -60,3 +60,8 @@ const (
 const (
 	ConnectionsIncrementor = KdDomainBase + "/connUpdateCounter"
 )
+
```

Comment:
nit: Please add a comment here clearly mentioning the difference between the 2 counters and why do we need both of them